### PR TITLE
Service Registration: Performance

### PR DESCRIPTION
**Pull request by kmh287**
This pull request distinguishes installing/registering the performance service and creating it.Please see #7839 for more context.

/to @choumx @jridgewell

### CODE REVIEW

File: test/functional/test-performance.js [hunk 1]
```diff
@@ -14,7 +14,11 @@
  * limitations under the License.
  */
 
-import {installPerformanceService} from '../../src/service/performance-impl';
+import * as sinon from 'sinon';
```
Comment (choumx) [inline test/functional/test-performance.js]: Nit: Unnecessary?

Comment (kmh287) [inline test/functional/test-performance.js]: When I merged master, the import was removed but there's still a reference to it on line 449:
https://github.com/kmh287/amphtml/blob/4853b2ce52c52b4f628c800b14da338bd486b7a6/test/functional/test-performance.js#L449

Comment (jridgewell) [inline test/functional/test-performance.js]: That's fine, sinon is globally installed in the test framework. We actually have a pending issue to remove all these imports.

Comment (kmh287) [inline test/functional/test-performance.js]: done

**Pull request merged**